kolibri/plugins/coach/assets/src/views/quizzes/QuizSummaryPage/index.vue

```diff
@@ -268,9 +268,14 @@
       },
       // @public
       setError(error) {
-        this.$store.dispatch('handleApiError', { error });
-        this.loading = false;
-        this.$store.dispatch('notLoading');
+        try {
+          this.$store.dispatch('handleApiError', { error });
```

Comment:
Should we only put the handleApiError call in here, so that the other two lines get executed?

Reply:
Updated here, thanks Richard